Admin.js

```diff
@@ -114,561 +136,771 @@ var Admin = {
 
+                filterToggler.
+                    removeClass("fa-check-square-o").
+                    addClass("fa-square-o");
                 target.hide();
 
             } else {
-                filterToggler
-                    .removeClass('fa-square-o')
-                    .addClass('fa-check-square-o')
-                ;
 
+                filterToggler.
+                    removeClass("fa-square-o").
+                    addClass("fa-check-square-o");
                 target.show();
+
             }
 
-            if (jQuery('div[sonata-filter="true"]:visible', filters_container).length > 0) {
+            if (jQuery("div[sonata-filter=\"true\"]:visible", filters_container).length > 0) {
+
                 jQuery(filters_container).slideDown();
+
             } else {
+
                 jQuery(filters_container).slideUp();
+
             }
+
         });
 
-        jQuery('.sonata-filter-form', subject).on('submit', function () {
-            jQuery(this).find('[sonata-filter="true"]:hidden :input').val('');
+        jQuery(".sonata-filter-form", subject).on("submit", function () {
+
+            jQuery(this).find("[sonata-filter=\"true\"]:hidden :input").
+                val("");
+
         });
 
         /* Advanced filters */
-        if (jQuery('.advanced-filter :input:visible', subject).filter(function () { return jQuery(this).val() }).length === 0) {
-            jQuery('.advanced-filter').hide();
-        };
+        if (jQuery(".advanced-filter :input:visible", subject).filter(function () {
+
+            return jQuery(this).val();
+
+        }).length === 0) {
+
+            jQuery(".advanced-filter").hide();
+
+        }
+
+        jQuery("[data-toggle=\"advanced-filter\"]", subject).click(() => {
+
+            jQuery(".advanced-filter").toggle();
 
-        jQuery('[data-toggle="advanced-filter"]', subject).click(function() {
-            jQuery('.advanced-filter').toggle();
         });
+
     },
 
     /**
      * Change object field value
      * @param subject
      */
-    set_object_field_value: function(subject) {
-        Admin.log('[core|set_object_field_value] set value field on', subject);
+    "set_object_field_value" (subject) {
+
+        Admin.log("[core|set_object_field_value] set value field on", subject);
+
+        this.log(jQuery("a.sonata-ba-edit-inline", subject));
+        jQuery("a.sonata-ba-edit-inline", subject).click(function (event) {
 
-        this.log(jQuery('a.sonata-ba-edit-inline', subject));
-        jQuery('a.sonata-ba-edit-inline', subject).click(function(event) {
             Admin.stopEvent(event);
-            var subject = jQuery(this);
+            const subject = jQuery(this);
             jQuery.ajax({
-                url: subject.attr('href'),
-                type: 'POST',
-                success: function(response) {
-                    var elm = jQuery(subject).parent();
+                "url": subject.attr("href"),
+                "type": "POST",
```

Comment:
- eslint: Expected object keys to be in ascending order. 'type' should be before 'url'.